articles/clients/index.md

```diff
@@ -4,186 +4,167 @@ description: Explains the basics of creating and using Auth0 Clients.
 
-    if (!userHasAccess) {
-      return callback(new UnauthorizedError('Access denied.'));
-    }
+- **Token Endpoint Authentication Method**: Defines the requested authentication method for the token endpoint. Possible values are `None` (public client without a client secret), `Post` (client uses HTTP POST parameters) or `Basic` (client uses HTTP Basic).
 
-    callback(null, user, context);
-}
-```
+- **Allowed Callback URLs**: Set of URLs to which Auth0 is allowed to redirect the users after they authenticate. You can specify multiple valid URLs by comma-separating them (typically to handle different environments like QA or testing). You can use the star symbol as a wildcard for subdomains (`*.google.com`). Make sure to specify the protocol, `http://` or `https://`, otherwise the callback may fail in some cases.
 
-## Auditing
+- **Allowed Logout URLs**: After a user logs out from Auth0 you can redirect them with the `returnTo` query parameter. The URL that you use in `returnTo` must be listed here. You can specify multiple valid URLs by comma-separating them. You can use the star symbol as a wildcard for subdomains (`*.google.com`). Notice that querystrings and hash information are not taking into account when validating these URLs. Read more about this at: [Logout](/logout).
 
-![](/media/articles/applications/applications-logs-auditing.png)
+- **Allowed Origins (CORS)**: Set of URLs that will be allowed to make requests from JavaScript to Auth0 API (typically used with CORS). This prevents same-origin policy errors when using Auth0 from within a web browser. By default, all your callback URLs will be allowed. This field allows you to enter other origins if you need to. You can specify multiple valid URLs by comma-separating them. You can use the star symbol as a wildcard for subdomains (`*.google.com`). Notice that querystrings and hash information are not taking into account when validating these URLs.
 
-The logs include many of the actions performed by the user:
+- **JWT Expiration (seconds)**: The amount of time (in seconds) before the Auth0 [JWT](/jwt) expires. The default value is `36000`, which maps to 10 hours.
 
-* Logging in to a Client;
-* Failing to log in to a Client;
-* Signing up;
-* Requesting a password change;
+- **Use Auth0 instead of the IdP to do Single Sign On**: If enabled, this setting prevents Auth0 from redirecting authenticated users with valid sessions to the identity provider (such as Facebook, ADFS, and so on).
 
-You can download the event logs using the [Management API](/api/management/v2#!/Logs/get_logs) or view them via the [Management Dashboard](${manage_url}/#/logs)
-
-While the [Management Dashboard](${manage_url}/#/logs) displays the log data in a neatly formatted manner, clicking on the row corresponding to a particular event displays the raw data, which looks something like this:
+::: panel-info Auth0 Rules
```

Comment:
I think it should stay. The doc is a generic introduction into Clients and rules are important. There was a whole paragraph here, which was too much, so I removed it and added this panel instead. 

It's the same with the logging info. I can remove the content to another doc, but I should mention it in this one so if anyone is interested can easily find this information.